diff -up pstoedit-4.00/QT/PstoeditQtGui/Makefile.am.omv~ pstoedit-4.00/QT/PstoeditQtGui/Makefile.am
--- pstoedit-4.00/QT/PstoeditQtGui/Makefile.am.omv~	2023-09-15 15:45:21.507659934 +0200
+++ pstoedit-4.00/QT/PstoeditQtGui/Makefile.am	2023-09-15 15:45:29.959677765 +0200
@@ -3,5 +3,5 @@ EXTRA_DIST+=resource.h pstoeditlogo-medi
 #EXTRA_DIST+=PstoeditQtGui.vcxproj PstoeditQtGui.vcxproj.filters PstoeditQtGui.vcxproj.user 
 
 GUI:    PstoeditQtGui.pro
-	qmake6 -o Makefile.qt PstoeditQtGui.pro
+	qmake-qt6 -o Makefile.qt PstoeditQtGui.pro
 	make -f Makefile.qt
diff -up pstoedit-4.00/QT/PstoeditQtGui/PstoeditQtGui.pro.omv~ pstoedit-4.00/QT/PstoeditQtGui/PstoeditQtGui.pro
--- pstoedit-4.00/QT/PstoeditQtGui/PstoeditQtGui.pro.omv~	2023-09-15 15:47:00.458846032 +0200
+++ pstoedit-4.00/QT/PstoeditQtGui/PstoeditQtGui.pro	2023-09-15 15:47:20.846274682 +0200
@@ -20,7 +20,7 @@ FORMS += \
 
 RESOURCES += PstoeditQtGui.qrc
 
-LIBS +=  -Wl,-rpath,`pwd`/../../../src/.libs/ -L ../../../src/.libs -lpstoedit
+LIBS +=  -Wl,-rpath,`pwd`/../../src/.libs/ -L ../../src/.libs -lpstoedit
 
 INCLUDEPATH += ../../src
 
